static/app/components/dropdownMenu.tsx

```diff
@@ -118,15 +118,20 @@ function DropdownMenu({
     [menuProps, hasFocus]
   );
 
+  const showDividers = useMemo(
+    () => stateCollection.some(item => !!item.props.details),
+    [stateCollection]
+  );
+
```

Comment:
Logical result of the two rules mentioned above: if _any_ item in the menu has some `details` text, then the entire menu should have dividers.